plugins/scaffolder-actions/scaffolder-backend-module-utils/src/actions/jsonata/yaml.ts

```diff
@@ -87,7 +96,11 @@ export function createYamlJSONataTransformAction(): TemplateAction<{
       if (ctx.input.as === 'object') {
         resultHandler = rz => rz;
       } else {
-        resultHandler = rz => YAML.stringify(rz, ctx.input.options);
+        if (ctx.input.writeMulti) {
```

Comment:
MINOR: A ternary would be a bit more concise here _and_ match the implementation in the handler below.